.github/workflows/perf.yml

```diff
@@ -147,28 +148,26 @@ jobs:
         with:
           path: artifacts
           
-      - name: Produce performance model table
-        run: python e2etest/get-performance-model-table.py
-      
-      - name: Create a new branch
-        uses: peterjgrainger/action-create-branch@v2.0.1
-        env:
-          GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
-        with:
-          branch: "perf-test-${{ github.run_id }}"
-          
-      - name: Commit to a branch
-        uses: stefanzweifel/git-auto-commit-action@v4
-        with:
-          commit_message: "Update benchmarking"
-          branch: "perf-test-${{ github.run_id }}"
-          file_pattern: docs/performance_model.md
-      
-      - name: pull-request
-        uses: repo-sync/pull-request@v2
+      - name: Produce performance report
+        run: python e2etest/get-performance-model-table.py ${{ needs.get-testing-version.outputs.testing_version }}
+
+      - name: Store benchmark result
+        uses: jefchien/github-action-benchmark@v1.11.12-alpha2
         with:
-          source_branch: "perf-test-${{ github.run_id }}"
-          destination_branch: "main"
-          github_token: ${{ secrets.GITHUB_TOKEN }}
-          pr_title: "Update Performance Model"
-          pr_body: "Generated by performance test workflow [#${{ github.run_number }}](https://github.com/aws-observability/aws-otel-collector/actions/runs/${{ github.run_id }}) using https://github.com/aws-observability/aws-otel-collector/commit/${{ needs.get-testing-version.outputs.commit_id }}."
+          tool: "customSmallerIsBetter"
+          output-file-path: performance-data.json
+          benchmark-data-dir-path: benchmark/trend
+          max-items-in-chart: ${{ env.MAX_BENCHMARKS_TO_KEEP }}
+          gh-pages-branch: ${{ env.GH_PAGES_BRANCH }}
+          github-token: ${{ secrets.GITHUB_TOKEN }}
+          commit-sha: ${{ github.event.inputs.sha }}
+          auto-push: false
```

Comment:
Can I ask what is this flag for?

Reply:
The `auto-push` flag, when set to true, will push the changes straight to the remote branch (https://github.com/benchmark-action/github-action-benchmark#auto-push-optional). Since we want to bundle the benchmark update with other changes, we need to disable the auto-push. It's false by default, so maybe it doesn't need to be explicitly set.

Reply:
Reasonable. I am fine with explicitly set this variable as false since I would know more about this command and its has more clarification with the flags though.